src/openrct2-ui/windows/Options.cpp

```diff
@@ -2280,6 +2281,7 @@ rct_window* window_options_open()
     w->page = WINDOW_OPTIONS_PAGE_DISPLAY;
     w->frame_no = 0;
     WindowInitScrollWidgets(w);
+    _hasFilePicker = OpenRCT2::GetContext()->GetUiContext()->HasFilePicker();
```

Comment:
You can probably revert the changes to this file now as you cache in the context instead.

Reply:
Wouldn't that increase the overhead again?

Reply:
The overhead is in the call that you have now cached. So this is going to be pretty cheap to call